04.class/class.js

```diff
@@ -0,0 +1,119 @@
+const fs = require('fs')
+const { MultiSelect } = require('enquirer')
+const { exit } = require('process')
+const minimist = require('minimist')(process.argv.slice(2))
+
+const reader = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+
+class Memo {
+  constructor () {
+    this.files = fs.readdirSync('./memo_data', 'utf-8')
+  }
+
+  getFirstLineAndPath () {
+    const result = this.files.map((path) => {
+      const fileContent = fs.readFileSync(`./memo_data/${path}`, 'utf-8')
+      const text = fileContent.split(/\r\n|\r|\n/)
+      const hash = { name: `${text[0]} in ${path}`, value: path }
+      return hash
+    })
+    return result
+  }
+
+  create () {
+    const inputs = []
+    reader.on('line', function (line) {
+      inputs.push('\n' + line)
+    })
+
+    reader.question('Make a note !\n', (answer) => {
+      inputs.push(answer)
+    })
+
+    reader.on('close', function () {
+      console.log('Your note was saved safely.')
+      const now = new Date()
+      fs.writeFileSync(`./memo_data/${now.getTime()}.txt`, inputs.join(''))
```

Comment:
```suggestion
      const timestamp = new Date().getTime()
      fs.writeFileSync(`./memo_data/${timestamp}.txt`, inputs.join(''))
```

みたいにした方が`writeFileSync`の呼び出しが少しシンプルになって良いかも。好みの問題ですが

Reply:
@JunichiIto さんレビューありがとうございます！

私もこちらの方がシンプルでいいなと考えましたのでいただいたように修正しました。
ご確認よろしくお願いします。
[fc6d617](https://github.com/OdenTakashi/js-practices/pull/4/commits/fc6d6170afe114f9f5e20458c93d8397183ddbfb)